actions/admin/site/update_advanced.php

```diff
@@ -17,28 +17,6 @@
 	throw new InstallationException("Passing a non-ElggSite to an ElggSite constructor!");
 }
 
-$site->url = rtrim(get_input('wwwroot', '', false), '/') . '/';
-
-elgg_save_config('path', sanitise_filepath(get_input('path', '', false)));
```

Comment:
`path` is no longer save, where does it come from if i ask `elgg_get_config('path')`?

I see `elgg_get_root_path()` already is based on different logic, was this documented?